app.py

```diff
@@ -21,41 +22,50 @@
     authority=app.config.get("AUTHORITY"),
     client_id=app.config["CLIENT_ID"],
     client_credential=app.config["CLIENT_SECRET"],
-    )
+)
+
 
 @app.route("/login")
 def login():
     return render_template("login.html", version=identity.__version__, **auth.log_in(
-        scopes=app_config.SCOPE,  # Have user consent scopes during log-in
-        redirect_uri=url_for("auth_response", _external=True),  # Optional. If present, this absolute URL must match your app's redirect_uri registered in Azure Portal
+        scopes=app_config.SCOPE, # Have user consent to scopes during log-in
+        redirect_uri=url_for("auth_response", _external=True), # Optional. If present, this absolute URL must match your app's redirect_uri registered in Azure Portal
```

Comment:
Changed it back to not store into result variable. The line lengths are long but they can delete the comments once they get it :)